app/javascript/modules/listingDetails/ListingDetailsEligibility.tsx

```diff
@@ -241,7 +241,38 @@ export const ListingDetailsEligibility = ({
             </InfoCard>
           </ListSection>
         )}
-        {!isHabitatListing(listing) && <ListingDetailsHMITable listing={listing} />}
+
+        {!isHabitatListing(listing) ? (
```

Comment:
Unless there was a good reason to change, let's stick with the `{!isHabitatListing(listing) && ...` style of conditional rendering, for consistency.